### PR TITLE
add an **experimental** transcript of the meeting 

### PR DESCRIPTION
**Pull request by ghinks**
I think it is a good idea to add a searchable transcript of the meeting in addition to the youtube video. I find it really useful.
This was created by [otter ai on my iphone](https://otter.ai/login)

### CODE REVIEW

**Comment by ljharb** [comment]
it doesn’t seem super accurate

File: meetings/transcripts/2019-08-27.transcript.txt [hunk 1]
```diff
@@ -0,0 +1,983 @@
...
+
+Darcy Clarke 22:28
+Okay, thanks for that overview. So The next thing on the agenda is open GSB 
+Foundation project progression.
+
+Glenn Hinks 22:38
+You put that on the agenda here?
+
+Darcy Clarke 22:49
+I think it was done. Dominic kiss, you added that.
```
Comment (dominykas) [inline meetings/transcripts/2019-08-27.transcript.txt]: I definitely prefer to be called "Dominic kiss" over "Glen Hinks" ;)

**Comment by Eomm** [comment]
What is the effort to create it?

File: meetings/transcripts/2019-08-27.transcript.txt (hunk 1 above)
Comment (ghinks) [reply meetings/transcripts/2019-08-27.transcript.txt]: haha. 
yes I'm still training the engine to recognize you. I think the machine processing defaulted to me every time there was not a good crisp sound.

**Comment by ghinks** [comment]
Yes it is not super accurate yet, and Im working on that. This time I just put my phone next to the laptop and turned on the app.
The effort is minimal. Meaning as long as I remember to turn it on we can get a transcript. I'm looking at running the application on osx next time.

File: meetings/transcripts/2019-08-27.transcript.txt (hunk 1 above)
Comment (ghinks) [reply meetings/transcripts/2019-08-27.transcript.txt]: I tagged you in a few places and told it to re-train. I think it has now identified you correctly ( sorry I did not tag you the first time around ).

**Comment by dominykas** [comment]
Do we have a plan for this or should we close it?

**Comment by Eomm** [comment]
It seems working only if one person take care of it, right now it doesn't seem auto-magic ✨

I'm pro to PR with this text to increase the searchable content and link it in the meeting's minute like youtube

**Comment by ghinks** [comment]
@Eomm I am willing to take this on. It is not perfect. However it makes searching for items much easier then you can always go back to the exact time spot in the video.

**Comment by styfle** [comment]
The caveat here is if the transcript is wrong and it makes it seem as if someone is saying something they are not. So as long as there is a disclaimer before the content, I think it would be great to have 👍

**Comment by Eomm** [comment]
> @Eomm I am willing to take this on. It is not perfect. However it makes searching for items much easier then you can always go back to the exact time spot in the video.

Great idea! If we define a "standard" sentence, like "tag discussion #issenumber", it would be possible to write the chapters list with the timing in the youtube video description!

Likes:
Issue #xxx 13:33
Issue #yyy 23:14
etc..

**Comment by ghinks** [comment]
If we can get a tag in the speech that would allow us to search and annotate. The actual speech to text conversion is super easy and takes place off line.

**Comment by wesleytodd** [comment]
I assume it is fine to let this close, but if you would like for this to move forward, please just re-target it to main.

**Pull request closed without merging**